test/test_nn.py

```diff
@@ -8983,18 +8983,17 @@ def _test_GroupNorm_cuda_half(self):
         output.sum().backward()
         self.assertEqual(output.type(), input.type())
 
-    def _test_module_empty_input(self, module, inp, check_size=True):
+    def _test_module_empty_input(self, module, inp, check_size=True, check_parameter_grads=True):
```

Comment:
I guess `check_parameter_grads` is an artefact of an older version? It does not seem to be used.

Reply:
It is not used, but it might be if someone adds support for new ops without taking care of gradients. I'll remove it for now though.